admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/handler/AdminMappingExceptionResolver.java

```diff
@@ -49,8 +49,9 @@ public ModelAndView resolveException(HttpServletRequest request, HttpServletResp
             
             mav.addObject("showDebugMessage", showDebugMessage);
             if (showDebugMessage) {
-                StringBuilder sb2 = new StringBuilder();
-                appendStackTrace(ex, sb2);
+                StringBuilder sb2 = new StringBuilder("An error was occured");
+                //uncomment next line to show stacktrace when error is occured
```

Comment:
maybe instead of commenting out add some property to show or not to show stacktrace ?